runtime/v2/runc/container.go

```diff
@@ -63,6 +64,15 @@ func NewContainer(ctx context.Context, platform rproc.Platform, r *task.CreateTa
 			Options: m.Options,
 		})
 	}
+
+	rootfs := ""
+	if len(mounts) > 0 {
+		rootfs = filepath.Join(r.Bundle, "rootfs")
+		if err := os.Mkdir(rootfs, 0711); err != nil {
```

Comment:
why not 700 or 755?

Reply:
We just moved that code here from https://github.com/containerd/containerd/blob/835e6d01fb46d197afdb76be6179ff7d9c75e3c4/runtime/v1/linux/bundle.go#L47-L49. Not sure why it is `0711` and if it is safe to change it.